core/controllers/feedback.py

```diff
@@ -117,13 +117,14 @@ class FeedbackStatsHandler(base.BaseHandler):
     """
 
     def get(self, exploration_id):
-        feedback_thread_analytics = feedback_services.get_thread_analytics(
-            exploration_id)
+        feedback_thread_analytics = (
```

Comment:
I don't understand why you need to to_dict() this. Wouldn't the original code work?